modules/lakeformation/main.tf

```diff
@@ -0,0 +1,34 @@
+#Granting all permissions to the emr_instance_profile_role and segment_datalake_role.
```

Comment:
Could you also add a space character after `#`?

Reply:
okay.